boto3/examples/cloudfront.rst

```diff
@@ -0,0 +1,37 @@
+Generate a signed URL for Amazon CloudFront
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following example shows how to generate a singed URL for Amazon CloudFront.
+Note that you will need the ``cryptography`` `library <https://cryptography.io/en/latest/>`__ to follow this example::
+
+    import datetime
+
+    from cryptography.hazmat.backends import default_backend
+    from cryptography.hazmat.primitives import hashes
+    from cryptography.hazmat.primitives import serialization
+    from cryptography.hazmat.primitives.asymmetric import padding
+    from botocore.signers import CloudFrontSigner
+
+
+    def rsa_signer(message):
+        with open('path/to/key.pem', 'rb') as key_file:
+            private_key = serialization.load_pem_private_key(
+                key_file.read(),
+                password=None,
+                backend=default_backend()
+            )
+        signer = private_key.signer(padding.PKCS1v15(), hashes.SHA1())
+        signer.update(message)
+        return signer.finalize()
+
+    key_id = 'APKBJCAOBHD36274ZIZA'
```

Comment:
Can you change this to the standard example key so that this doesn't trip up scanners like `git-secrets`: AKIAIOSFODNN7EXAMPLE